service/src/test/resources/application.yaml

```diff
@@ -114,9 +123,6 @@ tenant:
 
 okapi:
   url: ${OKAPI_URL:http://localhost:9130}
-  camunda:
-    base-path: /
-    rest-path: /rest
```

Comment:
What happened to the REST end points?

Reply:
It was moved to a default value in the [WorkflowEngineService.java](https://github.com/folio-org/mod-workflow/pull/117/files#diff-5f1399fae635ac57e696d503eb2135f7575b3a59a486e21613d3343f8f2a3243R50) and is where it should be updated to statically define workflow-engine in order for the module descriptor to match.

Otherwise, more variables will be required to template the module descriptor, and it will be chaos.